src/Codeception/Module/REST.php

```diff
@@ -841,6 +832,58 @@ public function seeResponseContainsJson($json = [])
         );
     }
 
+    /**
+     * Checks whether last response matches the supplied json schema
+     * Supply schema as json string
```

Comment:
Sorry for late response, but it isn't too late to modify this method.

As a new feature, this method would really benefit from more extensive documentation.
It would be good to add an example and some links to relevant documentation.

`$schema` is a large blob of JSON and you haven't used it inline even once, so it is very unlikely that someone else will.
It would be better to pass a path to schema file as a parameter (to be decided if it should be relative to project dir or data dir).
Codeception has a handy `codecept_absolute_path` function which leaves absolute paths as-is and appends relative paths to project_dir.